### PR TITLE
[ID] Add "User title"

### PR DESCRIPTION
**Pull request by Nivalyx**
Indonesian translation for the User title (http://osu.ppy.sh/wiki/en/User_Title) wiki article.

### CODE REVIEW

File: wiki/User_title/id.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  - gelar forum
+  - peringkat
+  - warna pengguna
+---
+
+# Gelar pengguna
+
+![Cuplikan laman profil Tom94](img/dev.png "Gelar \“osu!dev\” sebagaimana yang tersemat pada laman profil Tom94.")
+
+**Gelar pengguna** merupakan label atau tajuk khusus yang tersemat pada laman profil pemain. Suatu gelar pengguna pada umumnya disematkan oleh [osu!team](/wiki/People/The_Team) untuk menandakan status keanggotaan seseorang di dalam [gugus pengguna (*user group*)](/wiki/People/The_Team#user-groups) tertentu. Meskipun demikian, terdapat beberapa gelar yang [diberikan secara khusus](/wiki/People/Users_with_unique_titles) kepada pemain-pemain tertentu sebagai bentuk apresiasi atas pencapaian yang telah mereka raih dalam bidangnya masing-masing.
```
Comment (mousesalt) [inline wiki/User_title/id.md:13]:
```suggestion
**Gelar pengguna** merupakan label atau tajuk khusus yang tersemat pada laman profil pengguna. Suatu gelar pengguna pada umumnya disematkan oleh [osu!team](/wiki/People/The_Team) untuk menandakan status keanggotaan seseorang di dalam [gugus pengguna (*user group*)](/wiki/id/People/The_Team#user-groups) tertentu. Meskipun demikian, terdapat beberapa gelar yang [diberikan secara khusus](/wiki/id/People/Users_with_unique_titles) kepada pengguna-pengguna tertentu sebagai bentuk apresiasi atas pencapaian yang telah mereka raih dalam bidangnya masing-masing.
```

Comment (mousesalt) [reply wiki/User_title/id.md:13]: `Pengguna` lebih efisien digunakan daripada `Pemain`, banyak kontributor yang hanya melakukan registrasi dan tidak pernah memainkan game sama sekali.

File: wiki/User_title/id.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Gelar-gelar yang ada pada umumnya akan tertera dalam warna yang sesuai dengan warna gugus pengguna yang bersangkutan. Untuk gelar-gelar yang tidak terkait dengan keanggotaan suatu gugus pengguna, gelar yang bersangkutan akan tertera dalam warna putih.
```
Comment (mousesalt) [inline wiki/User_title/id.md:15]:
```suggestion
Gelar-gelar yang disematkan pada pengguna umumnya disesuaikan dengan warna gugus pengguna yang bersangkutan. Untuk gelar-gelar yang tidak terkait dengan keanggotaan gugus pengguna, gelar yang bersangkutan akan tertera dalam warna putih.
```

File: wiki/User_title/id.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+## Sejarah
+
+### Gelar-gelar yang terkait dengan jumlah postingan forum
+
+![Cuplikan tampilan profil Damnae pada forum osu! terdahulu](img/star-shooter.png "Gelar \“Star Shooter\” sebagaimana yang tersemat di atas jumlah postingan forum Damnae.
+")
+
+<!-- the bottom post on https://web.archive.org/web/20131107133826/https://osu.ppy.sh/forum/t/141240 -->
+
+Pada [situs web osu! yang lama](https://old.ppy.sh), terdapat beberapa gelar yang akan secara otomatis disematkan kepada para pemain yang telah menulis postingan forum dalam jumlah tertentu. Gelar-gelar ini hanya dapat dilihat pada laman forum dan tidak akan muncul pada laman profil masing-masing pemain.
```
Comment (mousesalt) [inline wiki/User_title/id.md:26]:
```suggestion
Pada [situs web osu! yang lama](https://old.ppy.sh), terdapat beberapa gelar yang akan secara otomatis disematkan kepada para pengguna yang telah menulis postingan di forum dalam jumlah tertentu. Gelar-gelar ini hanya dapat dilihat pada laman forum dan tidak akan muncul pada laman profil masing-masing pengguna.
```

**Review by mousesalt — COMMENTED** [review]
such a good translation

File: wiki/User_title/id.md (hunk 1 above)
Comment (Nivalyx) [reply wiki/User_title/id.md:13]: o you don't really need to add the locale (i.e. `wiki/id/...` or `wiki/fr/...`) in the translated article orz, the osu! website has been programmed to auto-recognize what language you're currently browsing the site/the wiki in :x

**Review by mousesalt — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**